compute_tools/src/http/api.rs

```diff
@@ -49,6 +50,11 @@ async fn routes(req: Request<Body>, compute: &Arc<ComputeNode>) -> Response<Body
             }
         }
 
+        (&Method::GET, "/num_cpus") => {
+            info!("serving /num_cpus GET request");
```

Comment:
maybe put the number here just because it's easy enough and it might be interesting.

Reply:
it makes sense, will add it